terraform/modules/daos_server/templates/startup_script.tftpl

```diff
@@ -38,4 +38,6 @@ systemctl start daos_server
 systemctl enable daos_agent
```

Comment:
Fine I will retest.  My testing need a few seconds of sleep on a single server for DMG commands to not error out.

Reply:
Please put the 5 seconds sleep back.  dmg cannot talk for a few seconds and the logic of startup gets messed up. 

May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: Created symlink from /etc/systemd/system/multi-user.target.wants/daos_agent.service to /u
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: + systemctl start daos_agent
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: + set -x
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: + echo 'BEGIN: DAOS server format'
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: BEGIN: DAOS server format
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: + [[ daos-server-0001 == \d\a\o\s\-\s\e\r\v\e\r\-\0\0\0\1 ]]
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: + dmg network scan
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: + grep --fixed-strings daos-server-0001
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: ERROR: dmg: 1 host had errors
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script:   daos-server-0001 the server at daos-server-0001:10001 refused the connection
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: + echo 'All DAOS Servers started'
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: All DAOS Servers started
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: + echo 'Formatting storage on servers: daos-server-0001'
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: Formatting storage on servers: daos-server-0001
May 13 06:56:04 daos-server-0001 google_metadata_script_runner[1371]: startup-script: + dmg storage format
May 13 06:56:05 daos-server-0001 google_metadata_script_runner[1371]: startup-script: Format Summary:
May 13 06:56:05 daos-server-0001 google_metadata_script_runner[1371]: startup-script:   Hosts            SCM Devices NVMe Devices
May 13 06:56:05 daos-server-0001 google_metadata_script_runner[1371]: startup-script:   -----            ----------- ------------
May 13 06:56:05 daos-server-0001 google_metadata_script_runner[1371]: startup-script:   daos-server-0001 1           2
May 13 06:56:05 daos-server-0001 google_metadata_script_runner[1371]: startup-script: + dmg system query -v
May 13 06:56:37 daos-server-0001 google_metadata_script_runner[1371]: startup-script: Rank UUID                                 Control Address   Fault Domain      State  Reas
May 13 06:56:37 daos-server-0001 google_metadata_script_runner[1371]: startup-script: ---- ----                                 ---------------   ------------      -----  ----
May 13 06:56:37 daos-server-0001 google_metadata_script_runner[1371]: startup-script: 0    d1745607-575d-4295-b4e8-37d60e60b11e 10.128.0.54:10001 /daos-server-0001 Joined
May 13 06:56:37 daos-server-0001 google_metadata_script_runner[1371]: startup-script:
May 13 06:56:37 daos-server-0001 google_metadata_script_runner[1371]: startup-script: + echo 'Done formating DAOS server'